core/src/test/java/com/backblaze/b2/client/B2LargeFileStorerTest.java

```diff
@@ -294,7 +354,7 @@ public Long getSrcLastModifiedMillisOrNull() {
         @Override
         public InputStream createInputStream() {
             // will not get called
-            throw new NotImplementedException();
+            return null;
```

Comment:
the comment says "will not get called".  since you changed it to not throw, i'm guesing that might not be true any more...?

Reply:
No there were some warnings about throwing that particular exception when I ran `gradlew test (or build- can't remember)` so I just changed it.